lib/lexicon.py

```diff
@@ -21,8 +22,8 @@ def __init__(self, orth=None, phon=None, synt=None, eval=None, special=None):
         :param Optional[list[str]] orth: list of spellings used in the training data
         :param Optional[list[str]] phon: list of pronunciation variants. Each str should
             contain a space separated string of phonemes from the phoneme-inventory.
-        :param Optional[list[list[str]]] synt: list of token sequences, for which each token sequence is used as as possible language model representation.
-            Each sublist should contain words from the language model vocabulary.
+        :param Optional[list[str]] synt: list of tokens, this token sequence
+            is used as as possible language model representation.
```

Comment:
Typo: "as as" -> "as a"

But since I'm on it may I suggest slight rewording to make clear that only one single token sequence is accepted.

```suggestion
        :param Optional[list[str]] synt: list of LM tokens that form a single token sequence.
            This sequence is used as the language model representation.
```

Reply:
👌 will do